src/404.html

```diff
@@ -87,17 +87,21 @@
 		<cap-nav></cap-nav>
 		<main id="main" class="l-interiorPage">
 			<header class="u-bg-gray-500 u-col-span-full">
-				<cap-page-header heading="Hmm... can't find that.">
+				<cap-page-header heading="Not Found.">
 					<p class="u-text-white u-text-serif">
-						Maybe the last borrower didn't return that link.
+						The Caselaw Access Project has updated and simplified our site to
+						provide long term, unrestricted access to U.S. case law.
```

Comment:
```suggestion
						provide long-term, unrestricted access to U.S. case law.
```

Reply:
Thank you. I should have tagged you in explicitly!